app/client/src/common/components/Page404.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @flow
+ */
+
+import React from 'react'
+
+function Page404() {
+    return (
+        <h1>
+            Error 404
+        <h1 />
+        <h2>
+            Oops, we are sorry but the page you are looking for was not found!
+            <br>
```

Comment:
With React's JSX, you need to have an explicit closing tag, even for self-closing elements. So you would need to change this to `<br />`.